setup/install_gatling/50_tune/10_puppet_infrastructure_tune.rb

```diff
@@ -1,25 +1,32 @@
 test_name 'Run puppet infrastructure tune' do
 
-  def run_puppet_infrastructure_tune
-    common_yaml = "/etc/puppetlabs/code-staging/environments/production/hieradata/common.yaml"
-
-    puts "Running 'puppet infrastructure tune' on master..."
+  def puppet_infrastructure_tune
+    base_tune_command = "puppet infrastructure tune"
+    common_yaml_path = "/etc/puppetlabs/code-staging/environments/production/hieradata/common.yaml"
+    tune_output_dir = "tune_output"
+    tune_output_path = "#{tune_output_dir}/nodes/#{master.hostname}.yaml"
+
+    # create tune_output dir
+    puts "Creating '#{tune_output_dir}' dir on master..."
     puts
+    on master, "mkdir -p #{tune_output_dir}"
 
-    output = on(master, "puppet infrastructure tune").output
-
-    # get data between '---' and the first empty line
-    data = output.match(/## Specify(.*)## CPU/m)[1].gsub("---", "").strip
-
-    # remove control codes
-    data = data.match(/0;32m(.*)\n.*\[/m)[1].strip + "\n"
+    # tune with hiera output and (and --force if specified)
+    hiera_option = " --hiera #{tune_output_dir}"
+    force_option = ENV["PUPPET_GATLING_SCALE_TUNE_FORCE"].eql?("true") ? " --force" : ""
```

Comment:
Looks like PUPPET_GATLING_SCALE_TUNE_FORCE has no default... should it?  And I assume force will cause it to override an existing file.  Any reason we wouldn't want to do that by default?

Reply:
This ensures that the command will only be run with the '--force' option if the PUPPET_GATLING_SCALE_TUNE_FORCE has specifically been set to 'true'. The '--force' option removes the enforcement of minimum system requirements. It has been added to allow testing of tuning smaller instances (m4.large, etc...). Nothing currently prevents overwriting the existing file.

Reply:
oh yeah.  I was thinking force was to overwrite the file...